lib/reporter/reporters/html.js

```diff
@@ -202,12 +202,21 @@ class HtmlReporter extends BaseReporter {
       }
     }
 
-    // Add skipped tests in completed section
-    for (const skippedTestName of module.skipped) {
+    // Add skipped tests by user in completed section
+    module.skippedByUser && module.skippedByUser.forEach(skippedTestName => {
       module.completedSections[skippedTestName] = {
-        status: 'skip'
+        status: 'skip',
+        userSkips: true
```

Comment:
```suggestion
        runtimeFailure: false
```

Reply:
What if there will be another type of skips, we will add in future? For e.g. environment skip or any argument specific skips. In my opinion, `userSkips` has the potential to adequately describe its usage in this context.

Thoughts?